wrappers/react/package.json

```
@@ -92,11 +91,11 @@
     "handsontable": ">=14.0.0"
   },
   "scripts": {
-    "build": "npm run clean && npm run build:commonjs && npm run build:es && npm run build:umd && npm run build:min",
-    "build:commonjs": "cross-env NODE_ENV=cjs rollup -c",
-    "build:umd": "cross-env NODE_ENV=umd rollup -c",
-    "build:es": "cross-env NODE_ENV=es rollup -c",
-    "build:min": "cross-env NODE_ENV=min rollup -c",
+    "build": "npm run clean && npm run build:es && npm run build:umd && npm run build:min",
```

```suggestion
    "build": "npm run clean && npm run build:commonjs && npm run build:es && npm run build:umd && npm run build:min",
```
After the changes, running `npm run build` does not build the dist files for CommonJS.

---

Thanks, accidentally removed during testing